libs/deal-ticket/src/components/deal-ticket/expiry-selector.tsx

```diff
@@ -22,6 +22,11 @@ export const ExpirySelector = ({ value, onSelect }: ExpirySelectorProps) => {
         onChange={(e) => onSelect(e.target.value)}
         min={minDate}
       />
+      {new Date() > date && (
```

Comment:
suggestion (non blocking): I'd probably pass a validation rule to the Controller component so that. See the rules prop here https://react-hook-form.com/api/usecontroller/controller/#main